### DISCUSSION TITLE
What versions should my packages for ace be on?

### DISCUSSION THREAD

**Discussion by AustinGitHub** [ajaxorg/ace#5102] [Q&A] [answered]
Hi, 
I am not sure what versions our packages should be on. 
We have 
`"ace-builds": "1.4.13"`
and
`"react-ace": "9.4.4"`

How do I know which version is the latest for `react-ace` as it seems `ace-builds` is no longer being updated (or it just pulls straight from this repo, I am unsure)

Thanks!

**Comment by nightwing** [comment, marked as answer]
latest versions of both packages should work fine together. See https://github.com/ajaxorg/ace-samples/tree/master/samples/react-ace-example

**Comment by AustinGitHub** [comment]
Thanks!
